content/Contributing/IssueReporting/Prerelease User Feedback.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Submitting Prerelease User Feedback"
+description: "How to submit feedback for individual pages in prerelease versions of TrueNAS software."
+weight: 15
+tags:
+- corecontributing
+- scalecontributing
+- coreissues
+- scaleissues
+- scalesupport
+---
+
+TrueNAS SCALE welcomes community feedback!
+
+To encourage general user experience reporting, prerelease versions of TrueNAS SCALE include the **How would you rate this page?** ![FeedbackIcon](/images/SCALE/23.10/FeedbackIcon.png "Feedback Icon") icon.
+Some screens undergoing revision also include a **NEW** badge and **Leave Feedback** link in the header.
+
+{{< trueimage src="/images/SCALE/23.10/NewLeaveFeedback.png" alt="NEW Badge and Leave Feedback Link" id="1: NEW Badge and Leave Feedback Link" >}}
+
+Click either the ![FeedbackIcon](/images/SCALE/23.10/FeedbackIcon.png "Feedback Icon") icon or **Leave Feedback** link to open a feedback widget that allows you to rate the web UI screen and leave comments.
```

Comment:
Widget is not correct, this is a window. Widgets apply to the system cards found on dashboards.
```suggestion
Click either the ![FeedbackIcon](/images/SCALE/23.10/FeedbackIcon.png "Feedback Icon") icon or **Leave Feedback** link to open a feedback window that allows you to rate the web UI screen and leave comments.
```